inspirehep/utils/record.py

```diff
@@ -107,6 +109,15 @@ def getitem(k, v, default):
     return value
 
 
+def get_record_type(record):
+    """Get record's type e.g. 'literature', 'jobs' etc."""
+    schema = record.get('$schema')
+    if schema:
+        return InspireRecordIdProvider.schema_to_pid_type(schema)
+    else:
+        return None
```

Comment:
```
else:
    return None
```
is not needed, because it will be returned anyway by the control flow reaching the end of the function.

Yet, is it really a sensible return value? I can already imagine the errors.